build/app.js

```diff
@@ -116,6 +117,11 @@ exports.execute = function(options) {
   var stageDir = utils.getFile(options.STAGE_DIR);
   utils.ensureFolderExists(stageDir);
 
+  var l10nTSPath = utils.joinPath(stageDir.path, 'timestamp-l10n.json');
+  if (utils.fileExists(l10nTSPath)) {
+    utils.deleteFile(l10nTSPath);
+  }
+
```

Comment:
Ricky, I decided to land it despite us not finalizing this conversation because I believe we can fix it in a follow up and I need to test it on master if we're going to try to land it on 2.2

But I'd like to make sure that this part of code is sound, so lets continue here.